tools/issue-labeler/src/IssueLabelerService/Answers/OpenAiAnswerService.cs

```diff
@@ -40,25 +40,63 @@ public async Task<AnswerOutput> AnswerQuery(IssuePayload issue, Dictionary<strin
             double scoreThreshold = double.Parse(_config.ScoreThreshold);
             double solutionThreshold = double.Parse(_config.SolutionThreshold);
 
-            var issues = await _ragService.SearchIssuesAsync(issueIndexName, issueSemanticName, issueFieldName, query, top, scoreThreshold, labels);
-            var docs = await _ragService.SearchDocumentsAsync(documentIndexName, documentSemanticName, documentFieldName, query, top, scoreThreshold, labels);
- 
-            if (docs.Count == 0 && issues.Count == 0)
+            //Subquery generation configurations
+            int subqueryCount = query.Length < 50 ? 3 : query.Length < 150 ? 5 : 7;
```

Comment:
For readability rather just do if else when its this nested.
You can also do something like divide by 15 with a max of 7 so we don't have these values hard coded but that's just me nitpicking. 
```suggestion
            const int charsPerSubquery = 15;
            const int maxSubqueries = 7;
            int subqueryCount = query.Length / querySubDivisions;
            subqueryCount = subqueryCount < maxSubqueries  ? subqueryCount : maxSubqueries;
            
```